Makefile

```diff
@@ -177,186 +184,214 @@ function cleanup { \
-.PHONY: uninstall
-uninstall: manifests kustomize
-	- $(KUSTOMIZE) build config/crd | $(CLUSTER_CLIENT) delete --ignore-not-found=$(ignore-not-found) -f -
+.PHONY: custom-scorecard-tests
+custom-scorecard-tests: fmt vet ## Build custom scorecard tests.
+	cd internal/images/custom-scorecard-tests/ && \
+	go build -o bin/cryostat-scorecard-tests main.go
 
-.PHONY: predeploy
-predeploy:
-	cd config/manager && $(KUSTOMIZE) edit set image controller=$(OPERATOR_IMG)
-	cd config/default && $(KUSTOMIZE) edit set namespace $(DEPLOY_NAMESPACE)
+.PHONY: scorecard-build
+scorecard-build: custom-scorecard-tests ## Build the custom scorecard OCI image.
+	printf '# Code generated by hack/custom.config.yaml.in. DO NOT EDIT.\n' > config/scorecard/patches/custom.config.yaml
+	envsubst < hack/custom.config.yaml.in >> config/scorecard/patches/custom.config.yaml
+# copy existing Dockerfile and insert --platform=${BUILDPLATFORM} into Dockerfile.cross, and preserve the original Dockerfile
+	sed -e '1 s/\(^FROM\)/FROM --platform=\$$\{BUILDPLATFORM\}/; t' -e ' 1,// s//FROM --platform=\$$\{BUILDPLATFORM\}/' internal/images/custom-scorecard-tests/Dockerfile > internal/images/custom-scorecard-tests/Dockerfile.cross
+ifeq ($(IMAGE_BUILDER), docker)
+	- $(IMAGE_BUILDER) buildx create --name project-v3-builder
+	$(IMAGE_BUILDER) buildx use project-v3-builder
+	- $(IMAGE_BUILDER) buildx build --push --platform=$(PLATFORMS) --tag $(CUSTOM_SCORECARD_IMG) -f internal/images/custom-scorecard-tests/Dockerfile.cross .
+	- $(IMAGE_BUILDER) buildx rm project-v3-builder
+else ifeq ($(IMAGE_BUILDER), podman)
+	BUILDAH_FORMAT=docker $(IMAGE_BUILDER) build -f internal/images/custom-scorecard-tests/Dockerfile.cross --manifest $(CUSTOM_SCORECARD_IMG) --platform $(PLATFORMS) . ; \
+	if [ "${MANIFEST_PUSH}" = "true" ] ; then \
+		$(IMAGE_BUILDER) manifest push $(CUSTOM_SCORECARD_IMG) $(CUSTOM_SCORECARD_IMG) ; \
+	fi
+else
+	$(error unsupported IMAGE_BUILDER: $(IMAGE_BUILDER))
+endif
+	rm internal/images/custom-scorecard-tests/Dockerfile.cross
 
-.PHONY: print_deploy_config
-print_deploy_config: predeploy
-	$(KUSTOMIZE) build config/default
+.PHONY: oci-build
+oci-build: manifests generate fmt vet test-envtest ## Build the OCI image.
+	BUILDAH_FORMAT=docker $(IMAGE_BUILDER) build --build-arg TARGETOS=$(OS) --build-arg TARGETARCH=$(ARCH) -t $(OPERATOR_IMG) .
 
-# Deploy controller in the configured Kubernetes cluster in ~/.kube/config
-.PHONY: deploy
-deploy: check_cert_manager manifests kustomize predeploy
-	$(KUSTOMIZE) build config/default | $(CLUSTER_CLIENT) apply -f -
-ifeq ($(DISABLE_SERVICE_TLS), true)
-	@echo "Disabling TLS for in-cluster communication between Services"
-	@$(CLUSTER_CLIENT) -n $(DEPLOY_NAMESPACE) set env deployment/cryostat-operator-controller-manager DISABLE_SERVICE_TLS=true
+# You need to be able to push the image for your registry (i.e. if you do not inform a valid value via OPERATOR_IMG=<myregistry/image:<tag>> than the export will fail)
```

Comment:
If using `podman` then you can set `MANIFEST_PUSH` to avoid this behaviour. I don't think there is an equivalent for `docker` though.

Reply:
Do you want me to change anything for this line?

Reply:
I suppose just explain about `MANIFEST_PUSH` with `podman`, rather than just saying "you need to be able to push" since that isn't always the case.